IHP/IDE/Postgres.hs

```diff
@@ -117,3 +118,15 @@ waitUntilReady process callback = do
     if "database system is ready to accept connections" `ByteString.isInfixOf` line
         then callback
         else waitUntilReady process callback
+
+waitUntilServerIsRunning :: (?context :: Context) => IO ()
+waitUntilServerIsRunning = do
```

Comment:
When using devenv we only need to wait until the Postgres server is running so when we drop support for the embedded Postgres server we can really strip down this module.